aopy/data/base.py

```diff
@@ -871,13 +873,6 @@ def align_recoring_drives(neuropixel_drive, drive2, subject):
     if subject not in subjects:
         raise ValueError('Subject not supported')
 
-    # Load ch map for drives
-    np_drive_ch_pos, recording_sites , _ = load_chmap(drive_type=neuropixel_drive)
-    if drive2 == 'Opto32': 
-        drive2_ch_pos, _ , _ = load_chmap(drive_type='ECoG244') # For opto drive we still want to center the neuropixel insert relative to ECoG
-    else:
-        drive2_ch_pos, _ , _ = load_chmap(drive_type=drive2)
-    
     # If drive2 is not a neuropixel drive, load alignment angle
     config_files = files('aopy').joinpath('config')
     if drive2 not in NP_drive_list:
```

Comment:
might want to make a note that this assumes a fixed mapping between subject and alignment which won't always be the case